kadi/commands/validate.py

```diff
@@ -733,7 +733,7 @@ def get_overlap_mask(times: np.ndarray, intervals: Table):
 
 
 @functools.lru_cache(maxsize=1)
-def get_command_sheet_exclude_intervals(state_key: str = None) -> Table:
+def get_command_sheet_exclude_intervals() -> Table:
```

Comment:
The `state_key` kwarg must be a leftover since the function does not use it and the call to this function does not provide it.